Tests/NIOHTTP2Tests/ContentLengthVerifierTests.swift

```diff
@@ -0,0 +1,27 @@
+import NIOHPACK
+@testable import NIOHTTP2
+import XCTest
+
+class ContentLengthVerifierTests: XCTestCase {
+    func testDuplicatedLengthHeadersPermitted() throws {
+        var headers = HPACKHeaders([("Host", "apple.com"), ("content-length", "1834"), ("User-Agent", "myCoolClient/1.0")])
+        XCTAssertNoThrow(try ContentLengthVerifier(headers))
```

Comment:
Can we test that the length is indeed parsed as `1834` and not nil or something else

Reply:
As far as I can tell this isn't easy with the currently exposed public information. I initially had a check which parsed the description but this seemed like a bad idea.

Reply:
You can consume 1384 bytes (which won't throw), and then consume 1 more (which will).

Reply:
We can make `expectedContentLength` `internal` instead of `private` and afterwards access it:
```swift
XCTAssertEqual(try ContentLengthVerifier(headers).expectedContentLength, 1834)
```